src/node_jsvmapi.h

```diff
@@ -110,6 +110,7 @@ NODE_EXTERN napi_status napi_create_symbol(napi_env e,
 NODE_EXTERN napi_status napi_create_function(napi_env e,
                                              napi_callback cb,
                                              void* data,
+                                             napi_propertyname name,
```

Comment:
Maybe also put the name parameter second to be consistent with the parameter order of `napi_define_class` ?